tests/common.py

```diff
@@ -443,9 +443,9 @@ def compare_forces(
             if compute_du_dx:
                 self.assert_equal_vectors(np.array(ref_du_dx), np.array(test_du_dx), rtol)
             if compute_du_dl:
-                np.testing.assert_almost_equal(ref_du_dl, test_du_dl, rtol)
+                np.testing.assert_allclose(ref_du_dl, test_du_dl, rtol=rtol)
```

Comment:
should we add an `atol` too?

Reply:
Do we want a `rtol` at all? Based on what is here: https://numpy.org/doc/stable/reference/generated/numpy.allclose.html it seems like `rtol` might be more flexible than we want? IE if the values are say 1,000,000 a relative tolerance of 1e-4 is 100, when we probably want to say that the values are always different by 0.001?

Reply:
> Do we want a rtol at all?

Using `atol` instead makes the most sense to me for the fixed-fixed comparisons, but also seems appropriate for fixed-floating like we have here. Maybe requiring both would make sense for fixed-floating?